src/main/webapp/app/course/manage/course-management.service.ts

```diff
@@ -223,6 +225,7 @@ export class CourseManagementService {
 @Injectable({ providedIn: 'root' })
 export class CourseExerciseService {
     private resourceUrl = SERVER_API_URL + `api/courses`;
+    private resourceUrlWithoutLocalSetup = SERVER_API_URL + `api`;
```

Comment:
Please do not mix up actual functionality and one for simulation. We should separate this as strictly as possible!

Reply:
I created a new service and moved the functionality there